zio-json/jvm/src/main/scala/zio/json/JsonDecoderPlatformSpecific.scala

```diff
@@ -43,21 +48,21 @@ trait JsonDecoderPlatformSpecific[A] { self: JsonDecoder[A] =>
    *
    * @see also [[decodeJsonStreamInput]]
    */
-  final def decodeJsonStream[R](stream: ZStream[R, Throwable, Char]): ZIO[R, Throwable, A] =
-    stream.toReader.use(readAll)
+  final def decodeJsonStream[R](stream: ZStream[R, Throwable, Char]): ZIO[R with Scope, Throwable, A] =
```

Comment:
This `with Scope` needs to be deleted (!)

Reply:
See #601 for fix.